lib/datasource/docker/common.ts

```diff
@@ -165,9 +169,14 @@ export function getRegistryRepository(
       if (!/^https?:\/\//.test(registryHost)) {
         registryHost = `https://${registryHost}`;
       }
+      let dockerRepository = lookupName.replace(registryEndingWithSlash, '');
+      const fullUrl = `${registryHost}/${dockerRepository}`;
+      const { origin, pathname } = parseUrl(fullUrl);
```

Comment:
Should we catch `null` returned by `parseUrl`? or simply buble null-ref up?

Reply:
We should ideally have validated `registryUrls` earlier, but I'm not sure if we do.